content/zh/docs/hertz/tutorials/basic-feature/protocol/http2.md

```diff
@@ -0,0 +1,80 @@
+---
+title: "HTTP2"
+date:  2022-12-10
+weight: 2
+description: >
+
+---
+HTTP2是HTTP协议的第二个版本，旨在通过改进第一个HTTP版本的许多缺点，使应用程序更快、更简单、更健壮。
+
+Hertz 提供了 HTTP2 的支持，参考了 [net/http2](https://github.com/golang/net/tree/master/http2) 的实现。
+
```

Comment:
Cloud you add the config doc of HTTP2? The server and client config is in https://github.com/hertz-contrib/http2/tree/main/config

Reply:
There is no need give an example for every config. Just like here: https://www.cloudwego.io/docs/hertz/reference/config/   . Add a table is enough